scss/civicrm/administer/civi-case/_case-types.scss

```diff
@@ -34,10 +34,23 @@
       }
 
       .ui-tabs-panel {
+        display: table;
+        overflow-x: auto;
         padding: 0 !important;
+        width: 100%;
 
-        table:first-child {
+        // Makes sure any element inside the panel has a true width 100% depending
+        // on the widest sibling and not the panel's width:
+        > * {
+          display: table-row;
+        }
+
+        * table:first-child {
           border-top: 0;
+
+          tr > td {
```

Comment:
```scss
tr {
 > td {
 }
}
```
or can't you simply do
```scss
td {

}
```
? Why is the `tr >` necessary?

Reply:
`tr >` was not necessary, it was just me being silly.